tests/SensioLabs/Connect/Tests/Api/ApiTest.php

```diff
@@ -27,14 +27,23 @@ class ApiTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->browser = $this->getMock('Buzz\\Browser');
-        $this->parser = $this->getMock('SensioLabs\\Connect\\Api\\Parser\\ParserInterface');
-        $this->logger = $this->getMock('Psr\\Log\\LoggerInterface');
+        $this->browser = $this->createAMock('Buzz\\Browser');
```

Comment:
What we are doing in Symfony is using `getMockBuilder()` and then call `getMock()` at the end.